app/spdb/build.sbt

```diff
@@ -427,7 +427,8 @@ def odbtest(version: Version) = AppConfig(
     "edu.gemini.dbTools.tcs.ephemeris.directory" -> "/home/software/ugemini/ephemerides",
     "edu.gemini.smartgcal.svnRootUrl"            -> "http://source.gemini.edu/gcal/branches/development/calibrations",
     "edu.gemini.spdb.dir"                        -> "/home/software/ugemini/spdb/spdb.active",
-    "edu.gemini.util.trpc.name"                  -> "Gemini ODB (Test)"
+    "edu.gemini.util.trpc.name"                  -> "Gemini ODB (Test)",
+    "org.osgi.framework.startlevel.beginning"    -> "50"
```

Comment:
Isn't the default startlevel 50? Why did you explicitly add it as such?

Reply:
The startlevel default is 99 everywhere else.  I dropped it to 50 for the test machines, which corresponds to the normal gemini bundles.  That leaves the reports bundle and the dataman off by default on the test machines.  In production and in development, reports runs by default but dataman still won't.